.idea/.name

```diff
@@ -0,0 +1 @@
+chromanews
```

Comment:
what is this ? ideally there should not be a .idea folder in repo

Reply:
not sure. how this was added. If we do, git add *, those will be ignored. Not sure if its added after performing git add *. I remember, we also need to update ignore file list.

Reply:
Updated .gitignore and now it is not uploading this.